cohorts/load.py

```diff
@@ -1297,6 +1297,10 @@ def compare_provenance(
     -----------
     Number of discrepancies (0: None)
     """
+    ## if either this or other items is null, return 0
```

Comment:
I guess you'll merge in the other PR and update this one, and this won't be here?